Client/game_sa/CEntitySA.cpp

```diff
@@ -678,6 +678,10 @@ bool CEntitySA::SetBoneRotation(eBone boneId, float yaw, float pitch, float roll
             RtQuat* boneOrientation = &frameData->m_pIFrame->orientation;
             RwV3d   angles = {yaw, roll, pitch};
             BoneNode_cSAInterface::EulerToQuat(&angles, boneOrientation);
+            CEntitySAInterface* theInterface = GetInterface();
+            if (theInterface) {
+                theInterface->bDontUpdateHierarchy = false;
+            }
```

Comment:
```cpp
if (GetInterface()) {
    GetInterface()->bDontUpdateHierarchy  = false;
}
```

Reply:
Thank you for noticing. I'll be carefull next time